backend/LexCore/Entities/MediaFile.cs

```diff
@@ -37,6 +37,7 @@ public class FileMetadata
     public int? SizeInBytes { get; set; }
     public string? FileFormat { get; set; }
     public string? MimeType { get; set; }
+    public string? LinkedFilesSubfolder { get; set; }
```

Comment:
we shouldn't be storing this here. The location of the file is stored in MediaFile

Reply:
Fixed.